app/main.py

```diff
@@ -1,7 +1,22 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+# app/main.py
+import os
+import requests
+
+# Define API base URL as a constant
+WEATHER_API_URL = "http://api.weatherapi.com/v1/current.json"
+
+
+def get_weather(api_key: str) -> None:
+    url = f"{WEATHER_API_URL}?key={api_key}&q=Paris"
+    response = requests.get(url)
+    response.raise_for_status()  # Raises an error for 4xx/5xx status codes
+    data = response.json()
+    print(f"Weather in Paris: {data['current']['condition']['text']}, {data['current']['temp_c']}°C")
 
 
 if __name__ == "__main__":
-    get_weather()
+    api_key = os.getenv("API_KEY")
```

Comment:
The `os.getenv("API_KEY")` function call is correct, but ensure that the environment variable `API_KEY` is set before running the script. Otherwise, the script will exit with an error message. Consider providing instructions on how to set this environment variable.